ajax/libs/just-animate/0.6.0/JustAnimate.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+var Helpers_1 = require('./core/Helpers');
+var Transformers_1 = require('./core/Transformers');
+var ElementAnimator_1 = require('./core/ElementAnimator');
+var SequenceAnimator_1 = require('./core/SequenceAnimator');
+var TimelineAnimator_1 = require('./core/TimelineAnimator');
```

Comment:
@kennynaoh Does this work in browsers? Especially you didn't commit a `core` folder.